buildSrc/src/main/groovy/com/mobbeel/plugin/task/CopyDependenciesTask.groovy

```diff
@@ -164,20 +165,24 @@ class CopyDependenciesTask extends DefaultTask {
      * @return
      */
     def processDependency(Dependency dependency, String archiveName, String dependencyPath) {
+		println "files for dep: ${project.fileTree(dependencyPath).getFiles()}"
         project.fileTree(dependencyPath).getFiles().each { file ->
-            if (file.name.endsWith(".pom")) {
+            if (dependency.group in packagesToExclude) {
+				println "dependency in exclusion list"
+			} else if (file.name.endsWith(".pom")) {
                 println "POM: " + file.name
                 processPomFile(file.path)
             } else {
                 if (archiveName == null || file.name == archiveName) {
                     println "Artifact: " + file.name
                     if (file.name.endsWith(".aar")) {
-                        processZipFile(file, dependency)
+                        processZipFile(file, dependency.name, dependency.group, dependency.version)
+						println "   |--> zip processed"
```

Comment:
There's excessive tabulation

Reply:
Sorry, must have set wrong tab/space config in one of my editors